common/memtest.py

```diff
@@ -24,8 +26,16 @@ def logMemory(cls, section_name):
             MemTracker.heapy_instance = hpy()
             MemTracker.heapy_instance.setrelheap()
         
-        h = MemTracker.heapy_instance.heap()
         with open(MemTracker.log_file_path, "a") as f:
-            f.write("Memory statistics for '" + section_name + "':\n")
-            f.write(str(h) + "\n")
-            f.write("\n")
+            f.write("Memory statistics for '" + section_name + "':\n\n")
+            start = dt.datetime.now()
+            f.write("Starting TimeStamp: " + str(start))
+            f.write("\n\n")
```

Comment:
I personally feel like logging new lines only clutters our output, especially after we have the nicely formatted logs from python's logging library. Let me know if you feel strongly otherwise.